build/tests/shell-test.psh

```diff
@@ -18,78 +18,17 @@ set --conditional -s data -v .
-service_db add --url http://${service_host}:7103 --name es7103 --type eservice
-service_db add --url http://${service_host}:7104 --name es7104 --type eservice
-service_db add --url http://${service_host}:7105 --name es7105 --type eservice
-
-## default pservice group
-pservice add --url http://${service_host}:7001
-pservice add --url http://${service_host}:7002
-pservice add --url http://${service_host}:7003
-
-## pservice group p1
-pservice add --group p1 --url http://${service_host}:7003
-pservice add --group p1 --url http://${service_host}:7004
-pservice add --group p1 --url http://${service_host}:7005
-
-## pservice group all
-pservice add --group all --url http://${service_host}:7001
-pservice add --group all --url http://${service_host}:7002
-pservice add --group all --url http://${service_host}:7003
-pservice add --group all --url http://${service_host}:7004
-pservice add --group all --url http://${service_host}:7005
-
-## default eservice group
-eservice add --url http://${service_host}:7101
-eservice add --url http://${service_host}:7102
-eservice add --url http://${service_host}:7103
-
-## eservice group e1
-eservice add --group e1 --name es7103 es7104 es7105
-eservice use --group e1 --url http://${service_host}:7105
-
-## eservice group all
-eservice add --group all --url http://${service_host}:7101
-eservice add --group all --url http://${service_host}:7102 http://${service_host}:7103 --name es7104 es7105
-eservice use --group all --name es7104
-
-## default sservice group
-set -s persistent_storage_service -v http://${service_host}:7201
-
-sservice add --url http://${service_host}:7201
-sservice add --url http://${service_host}:7202
-sservice add --url http://${service_host}:7203
-sservice set --duration 120 --replicas 2 --persistent ${persistent_storage_service}
-
-## sservice group s1
-sservice add --group s1 --url http://${service_host}:7203
-sservice add --group s1 --url http://${service_host}:7204
-sservice add --group s1 --url http://${service_host}:7205
-sservice set --group s1 --duration 120 --replicas 2 --persistent ${persistent_storage_service}
-
-## sservice group s2
-sservice add --group s2 --url http://${service_host}:7201
-sservice add --group s2 --url http://${service_host}:7202
-sservice set --group s2 --duration 120 --replicas 1 --persistent ${persistent_storage_service}
-
-## all sservices
-sservice add --group all --url http://${service_host}:7201
-sservice add --group all --url http://${service_host}:7202
-sservice add --group all --url http://${service_host}:7203
-sservice add --group all --url http://${service_host}:7204
-sservice add --group all --url http://${service_host}:7205
-sservice set --group all --duration 3600 --replicas 3 --persistent ${persistent_storage_service}
+## some definitions to make it easier to display text
+set -s ENDC   -v "\033[0m"
+set -s BOLD   -v '\033[1m'
+set -s HEADER -v "\033[95m"
+set -s ERROR  -v "\033[91m"
+set -s WARN   -v "\033[93m"
+set -s INFO   -v "\033[92m"
```

Comment:
We seem to have the same definition at multiple places: worth to "outsource" into a shared library?

Reply:
yes and no. yes we should. no we're not going to in the short term. create an issue for it. but... pdo-shell is not *that* good at pulling in "libraries".